metabase/metabase
# feat(sdk): add background-disabled color

Closes https://github.com/metabase/metabase/issues/47559

### Description

Adds the `colors.background-disabled` color for disabled element's background color.

### How to verify

- Set the `colors.background-disabled` color in the theme options
- Render a disabled button, such as the save button using `<InteractiveQuestion isSaveEnabled />`, which should be in a disabled state by default
- The disabled color should change

### Demo

The default save button's color now changes.

![CleanShot 2567-11-13 at 01 11 04@2x](https://github.com/user-attachments/assets/0f48a33a-1eb3-4da8-8ded-7d94bba991ac)

### Checklist
